models/EmailWhitelist.php

```diff
@@ -100,13 +100,18 @@ public static function getWhitelists()
         public static function emailIsAllowed($email)
         {
                 $allowed = self::getWhitelists();
-		$domain = explode('@', $email);
+		        $domain = explode('@', $email);
```

Comment:
@katka-n Please configure your IDE to use only spaces for indenting code and pls fix formatting of this file. 
If that takes 5 mins, than pls possibly create editor config file - http://editorconfig.org/ with a PHP set up so anyone modifying the code formats the code the same way.